Example/AlloyCodelessLiteiOSDemo/ContentView.swift

```diff
@@ -34,11 +34,15 @@ struct ContentView: View {
                             // *** this key is part of a working example ***
                             // You should obtain your key from the alloy dashboard
                             // On settings > SDK Config
-                            apiKey: "9ca83767-f213-4aaf-bc1b-1ed0a89eaf23", 
+                            apiKey: "22501763-79c3-4f8d-a069-ce99afad16d5",
                             production: false,
                             realProduction: true,
                             codelessFinalValidation: false,
-                            showDebugInfo: true
+                            showDebugInfo: true,
+                            journeyToken: "J-UMEhLDP3p759425pz1uP",
+                            journeyApplicationToken: "JA-wXLDZMFCnKIor5xqaOcH",
+                            appUrl: "https://alloysdk.alloy.co",
+                            apiUrl: "https://docv.alloy.co"
```

Comment:
@jasoncuriano will the new routes will be available without removing this?

Reply:
yes! both old and new url cnames are added in corekube-prod now

Reply:
new I get this:

```
HEADERS:
["Content-Type": "application/json"]

METHOD:
POST

URL:
https://corekube-prod-alloysdk.api.alloy.com/auth/init?production=false

BODY DATA:
{
  "id" : "22501763-79c3-4f8d-a069-ce99afad16d5",
  "journey_application_token" : "JA-wXLDZMFCnKIor5xqaOcH",
  "journey_token" : "J-UMEhLDP3p759425pz1uP"
}

RESPONSE DATA:
{"status_code":400,"error":{"minor_code":4001,"type":"Header Validation Failure","message":"Verify that your OAuth bearer token is well formed and that your application version is a positive integer if passed.","details":{"message":"\"authorization\" is required","path":"authorization","type":"any.required","context":{"key":"authorization"}},"error_token":"e8319361-7fd6-4933-bdf4-321a1fb11d78"},"timestamp":1733252319771,"evaluation_token":null,"entity_token":null,"application_token":null,"application_version_id":null}

```

Reply:
maybe we are missing a rule?